versions/t-/tfc-framework.json

```diff
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "2436086545723f9d673b8b5b900902e2f5d87167",
+      "git-tree": "6b86673d223034a66c1646bb7f8b62ab1caa0c75",
```

Comment:
Can we populate this automatically? With this hand picked it is not certain that
you are installing the code from you current commit.